master/internal/core.go

```diff
@@ -908,10 +908,6 @@ func (m *Master) Run(ctx context.Context) error {
 	searcherGroup := m.echo.Group("/searcher", authFuncs...)
 	searcherGroup.POST("/preview", api.Route(m.getSearcherPreview))
 
-	trialsGroup := m.echo.Group("/trials", authFuncs...)
-	trialsGroup.GET("/:trial_id", api.Route(m.getTrial))
-	trialsGroup.GET("/:trial_id/metrics", api.Route(m.getTrialMetrics))
```

Comment:
I did end up finding out `get t describe` uses this still.

Reply:
thanks. I'll see if the tests flag this, then see if the Trials CLI can be changed (will look in +1 week or so?)

Reply:
yeah. updating the trial CLI would be nice, anyway. though it is work.

Reply:
I've replaced the e2e_tests use of legacy trials API, so I think this is about to pass tests 🤞